lk2nd/smb1360/smb1360-qcom.c

```diff
@@ -15,21 +22,105 @@ static const struct smb1360_battery batteryB = {
 	.profile = SMB1360_BATTERY_PROFILE_B,
 };
 
+static const struct smb1360_battery batteryK = {
+	.name = "Default Profile",
+};
+
+static int64_t abs64(int64_t val)
+{
+	return val < 0 ? (-1 * val) : val;
+}
+
+static int smb1360_check_batt_profile(const void *fdt, int offset, uint32_t connected_rid)
+{
+	const uint32_t *prop;
+	status_t ret;
+
+	int i, len, timeout, loaded_profile, new_profile;
+	uint8_t val = 0;
+	uint32_t profile_rid[BATTERY_PROFILE_MAX], rid10;
+
+	prop = fdt_getprop(fdt, offset, "qcom,profile-a-rid-kohm", &len);
+	if (len < 0)
+		return -EINVAL;
+	profile_rid[0] = fdt32_to_cpu(*prop);
```

Comment:
maybe make this normal variables now and call it

```suggestion
	profile_a_rid = fdt32_to_cpu(*prop);
```

Reply:
Sure, I'll change it now